src/panes.ts

```diff
@@ -52,6 +51,7 @@ export type SearchAndSelectPane = Pane<
     }>
     selection_mode?: "none" | "single" | "multiple"
     context?: "device"
+    manufacturer_name?: string
```

Comment:
How is this used? Is it possible for it to be part of a more generic property? Or is selecting devices a specific-enough use case that it should be its own pane?

Reply:
Unfortunately I don't have your typescript wizardry, I wanted to make it such that the other props are a function of context value. But for the use-case atm it's too simple, so no need to overengineer the type

Reply:
Sorry, I was more asking where `manufacturer_name` is displayed. If we start to have a lot of props specific to this one screen, I think we should make it its own pane.

Reply:
Can be optimised later. But since this won't see any further changes I think we're good for now. Can create a low prio issue for this.